common/app/model/EmailAddons.scala

```diff
@@ -453,6 +453,11 @@ case object AustraliasModernOutback extends FrontEmailMetadata {
   override val banner = Some("aus-modern-outback.png")
 }
 
+case object PushingButtons extends FrontEmailMetadata {
+  val name = "Pushing Buttons"
```

Comment:
You may already know more, but my vague memory is that this name has to match one of the fields from the fronts tool where this front is set up. I imagine it's probably good as-is, but thought I'd mention anyway!